frontend/app/(tabs)/create.tsx

```diff
@@ -13,42 +14,66 @@ import {
   H5,
   Text,
   TextArea,
-  View,
+  Switch,
 } from 'tamagui';
-import type { TabsContentProps } from 'tamagui';
 
 export default function Create() {
   const [isEnabled, setEnabledElements] = useState(false);
 
   return (
     <View>
       <YStack>
-        <H4 style={{ alignSelf: 'center' }}>NEW POST</H4>
-        <Tabs
-          defaultValue='tab1'
-          orientation='horizontal'
-          flexDirection='column'
-          width={400}
-          height={150}
-          borderRadius='$4'
-          borderWidth='$0.25'
-          overflow='hidden'
-          borderColor='$borderColor'
-        >
-          <Tabs.List
-            disablePassBorderRadius='bottom'
-            aria-label='Creating a New Post'
+        <H4 style={{ alignSelf: 'center', color: 'black' }}>NEW POST</H4>
```

Comment:
For the H4 styling if you just put the color `black`, it will show on both `light` and `dark` mode and that doesn't make sense to have it be `black` if we are on `light` mode. I think there is a value provided called `$accentColor` or `$accentBackground` that should fix this issue.